content/post/mnist-example.md

```diff
@@ -0,0 +1,155 @@
++++
+title = "A Scikit-learn pipeline in Wallaroo"
+draft = false
+date = 2017-11-16T00:00:00Z
```

Comment:
Date needs to be corrected. I mention it more because it took me a while to find the post on the netlify preview then anything else (I was looking for the preview at the top of the site with the newest posts).